roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
```

Comment:
I really like this and looking forward to be handling atomic. However, could that be a follow-up PR? (and issue?)